pom.xml

```diff
@@ -9,9 +9,42 @@
         <version>9</version>
     </parent>
 
-    <groupId>uk.co.qualitycode</groupId>
+    <groupId>io.github.excitedkettle</groupId>
     <artifactId>functional-utils</artifactId>
-    <version>2.0-SNAPSHOT</version>
+    <version>2.0</version>
+    <packaging>jar</packaging>
+
+    <name>Functional Utils</name>
+    <description>A utility library for adding functional interfaces to core java 8+</description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Richard Polton</name>
+            <organizationUrl>https://github.com/rpolton</organizationUrl>
+        </developer>
+    </developers>
+    <contributors>
+        <contributor>
+            <name>Ryan Wills</name>
+            <email>rpwills123@live.co.uk</email>
+            <organizationUrl>https://github.com/ExcitedKettle</organizationUrl>
+        </contributor>
+    </contributors>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>central</id>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Currently it is only going to a snapshot repository, so will need to make further changes to have snapshot + release builds.